### DISCUSSION TITLE
Portuguese translation, maybe its a good idea

### DISCUSSION THREAD

**Discussion by horacio3m** [mikaelacaron/Basic-Car-Maintenance#209] [Ideas]
I'm Brazilian, what do you think about adding Portuguese to the app? I noticed that this language hasn't been added yet. If you'd like, I'd love to work on this issue.
<img width="1087" alt="Captura de Tela 2023-10-23 às 22 34 22" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/8496232/58956901-75f2-487b-9c5a-75db89b672c7">

**Comment by mikaelacaron** [comment]
Yes go ahead! This is covered under Issue #7 anyone is welcome to add any language that isn't there
